src/com/maddyhome/idea/vim/helper/TagBlockHelper.java

```diff
@@ -0,0 +1,215 @@
+package com.maddyhome.idea.vim.helper;
+
+import com.intellij.lexer.Lexer;
+import com.intellij.lexer.LexerPosition;
+import com.intellij.lexer.XmlLexer;
```

Comment:
I use the jetbrains XML lexer to parse tags. I assumed that it would be no problem but idk what is the politics about jetbrains platform dependencies.